apps/kampus/app/pano/PanoFeed.tsx

```diff
@@ -0,0 +1,77 @@
+import { Suspense, useCallback } from "react";
+import { graphql, usePaginationFragment } from "react-relay";
+
+import { Button } from "@kampus/ui-next";
+
+import { PostItem } from "~/app/pano/features/post-list/PostItem";
+import { type PanoFeedFragment$key } from "./__generated__/PanoFeedFragment.graphql";
+
+const fragment = graphql`
+  fragment PanoFeedFragment on Viewer
```

Comment:
This is probably going to be the most complex fragment we'll have to write, but go read the docs for it: https://relay.dev/docs/tutorial/connections-pagination/